classes/Entities/BaseModel.php

```diff
@@ -117,6 +117,22 @@ public function save(array $options = [])
         return $saved;
     }
 
+    /**
+     * Update the model in the database.
+     *
+     * @param  array  $attributes
+     * @param  array  $options
+     * @return bool
+     */
+    public function update(array $attributes = [], array $options = [])
```

Comment:
type hint ': bool' missing

Reply:
Ive just copied as is from the eloquent update() method. I would prefer to leave it off for now if you dont mind as im not sure if adding a strict return type will create an issue elsewhere.